examples/create_identity_provider/github/main.tf

```diff
@@ -34,6 +34,6 @@ resource "rhcs_identity_provider" "github_idp" {
   github = {
     client_id     = var.github_client_id
     client_secret = var.github_client_secret
-    organizations = var.github_orgs
+    organizations = [var.github_org]
```

Comment:
wdym by saying the array isn't being passed to terraform? we should have some examples like that in assisted I think: https://github.com/openshift/assisted-test-infra/blob/ca8ab6295b6e401508d2d164058bdb2e58e4ab93/terraform_files/baremetal/main.tf#L26

Reply:
I think its something to do with the **(OLD)** sdk version we are using. An old one.

We could not figure it out. So we will reach out to HashiCorp. But for now, we want the example to simply work.